.github/CONTRIBUTING.md

```diff
@@ -1,14 +1,14 @@
-:wave: Welcome! We'd love to accept your patches and contributions to this project. 
+:wave: Welcome! We'd love to accept your patches and contributions to this project.
 
 ## How can I contribute?
 
 - [Suggest](https://github.com/google/blockly-samples/issues/new?assignees=&labels=type%3A+feature+request%2C+triage&template=feature_request.md) plugins, examples, or codelabs
-  - :arrow_right: Start by reading [How To Write A Good Issue](https://github.com/google/blockly-samples/wiki/How-to-Write-a-Good-Issue)
+  - :arrow_right: Start by reading [How To Write A Good Issue](https://developers.google.com/blockly/guides/modify/contribute/write_a_good_issue)
 - Implement plugins, examples, or codelabs
-  - :arrow_right: Read up on how to [Add a Plugin](https://github.com/google/blockly-samples/wiki/Adding-A-Plugin) or [Write a Codelab](https://github.com/google/blockly-samples/wiki/Writing-a-Codelab)
-- [Report bugs](https://github.com/google/blockly-samples/issues/new?assignees=&labels=type%3A+bug%2C+triage&template=bug_report.md)  
-  - :arrow_right: Start by reading [How To Write A Good Issue](https://github.com/google/blockly-samples/wiki/How-to-Write-a-Good-Issue)
-- Fix bugs
+  - :arrow_right: Read up on how to [Add a Plugin](https://developers.google.com/blockly/guides/modify/contribute/add_a_plugin) or [Write a Codelab](https://developers.google.com/blockly/guides/modify/contribute/write_a_codelab)
+- [Report bugs](https://github.com/google/blockly-samples/issues/new?assignees=&labels=type%3A+bug%2C+triage&template=bug_report.md)
+  - :arrow_right: Start by reading [How To Write A Good Issue](https://developers.google.com/blockly/guides/modify/contribute/write_a_good_issue)
+- [Fix bugs](https://developers.google.com/blockly/guides/modify/contribute/issue_labels)
```

Comment:
This link should be on `Finding a Good Issue` below instead. Currently that one still links to the wiki. It should link to dev site instead and `Fix bugs` shouldn't have a link.